frontend/app/components/catList.jsx

```diff
@@ -0,0 +1,27 @@
+import React from 'react'
+import CatCard from './catCard'
+import { Grid, Row } from 'react-bootstrap'
+
+class CatList extends React.Component {
+  constructor(props) {
+    super(props)
+  }
+
+  render() {
```

Comment:
You need some loading behavior on this view: basically a function that shows a loading GIF until the data is ready. You can see what I mean if you do the following:

1) search for a zip code, watch the list render
2) go back to the main screen
3) search for a different zip code
4) it will show the results from step 1 for a half second and then replace them with the results from step 3. you can avoid this with a loading GIF

I'm sure `React` has its own special way to do that, but it should be easy to look up. It should happen either here or on the `listContainer` component, your choice.

Reply:
Yup, i'll write separate tickets for handling this, and for handling error states (ie invalid zip code, cat not found etc)